ldap-client.go

```diff
@@ -202,3 +202,28 @@ func (lc *LDAPClient) GetGroupsOfUser(username string) ([]string, error) {
 	}
 	return groups, nil
 }
+
+// GetLdapGroups returns list of groups matching a name.
+func (lc *LDAPClient) GetLdapGroups(groupName string) ([]string, error) {
+	err := lc.Connect()
+	if err != nil {
+		return nil, err
+	}
+
```

Comment:
please add here the bind block:
// First bind with a read only user
	if lc.BindDN != "" && lc.BindPassword != "" {
		if err = lc.Conn.Bind(lc.BindDN, lc.BindPassword); err != nil {
			return
		}
	}

Reply:
added